openedx/core/djangoapps/content/course_overviews/tasks.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from celery import task
+from celery_utils.logged_task import LoggedTask
+from celery_utils.persist_on_failure import PersistOnFailureTask
+from django.conf import settings
+
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_ALL_COURSES = False
+
+DEFAULT_CHUNK_SIZE = 50
+
+DEFAULT_FORCE_UPDATE = False
```

Comment:
Nittiness: is the extra vertical spacing necessary?

Reply:
Oh, hmm, I thought that was a thing?  Possibly just personal habit...